src/table/Exposure.cc

```diff
@@ -417,7 +384,7 @@ PTR(ExposureTable) ExposureTable::make(Schema const & schema) {
             "Schema for Exposure must contain at least the keys defined by makeMinimalSchema()."
         );
     }
-    return std::make_shared<ExposureTableImpl>(schema);
+    return std::shared_ptr<ExposureTable>(new ExposureTable(schema));
```

Comment:
The constructor is protected, and I do not know how to make `make_shared` a friend (see also conversation with @kfindeisen elsewhere on this PR).